docs/howtos/cluster-resource-override.md

```diff
@@ -0,0 +1,208 @@
+# How-to Guide: Using the Fleet `ClusterResourceOverride` API
+
+This guide provides an overview of how to use the Fleet `ClusterResourceOverride` API to override cluster resources.
+
+## Overview
+`ClusterResourceOverride` is a feature within Fleet that allows for the modification or override of specific attributes 
+across cluster-wide resources. With ClusterResourceOverride, you can define rules based on cluster labels or other 
+criteria, specifying changes to be applied to various cluster-wide resources such as namespaces, roles, role bindings, 
+or custom resource definitions. These modifications may include updates to permissions, configurations, or other 
+parameters, ensuring consistent management and enforcement of configurations across your Fleet-managed Kubernetes clusters.
+
+## API Components
+The ResourceOverride API consists of the following components:

```

Comment:
Should it be ClusterResourceOverride API not ResourceOverride API?

Reply:
yep this looks like a typo